.travis.yml

```diff
@@ -1,20 +1,42 @@
+sudo: required
+dist: xenial
 language: go
 go: "1.10"
 
 services:
 - docker
 
-before_script:
-- go get -u github.com/golang/dep/cmd/dep
-- echo "$NFDPASS" | docker login -u="$NFDNAME" quay.io --password-stdin
+branches:
+  only:
+  - master
 
+addons:
+  apt:
+    sources:
+    - sourceline: 'ppa:ansible/ansible'
+    packages:
+    - ansible
+    - jq
 
 jobs:
   include:
-  - stage: Build Image
-    name: build kubevirt-node-labeller image
-    script: make image 
+  - stage: Functional test
+    name: Test kubevirt-node-labeller bundle
+    before_script:
+    - git clone https://github.com/fabiand/traviskube.git
+    - mv traviskube ci 
+    - sudo mount --make-rshared /
+    - bash -x ci/prepare-host minishift
+    - bash -x ci/prepare-host oc
+    - bash -x ci/start-cluster minishift
+    - bash -x ci/deploy-kubevirt minishift v0.16.0
```

Comment:
There is 0.17.0 now I think

Reply:
But do you need kubevirt at all? I mean this is an isolated labeller component so checking whether the nodes got the labels might be enough..

Reply:
you are right, Since travis doesn't have access to kvm, so kubevirt is not necessary

Reply:
for some unknown reasons, libvirt doesn't show any output of domcapabilities without kubevirt. I will have to investigate it.